contracts/src/starknet/rewaave/tokens/ETHstaticAToken.cairo

```diff
@@ -140,9 +147,16 @@ func burn{syscall_ptr : felt*, pedersen_ptr : HashBuiltin*, range_check_ptr}(
 end
 
 func claim_rewards{syscall_ptr : felt*, pedersen_ptr : HashBuiltin*, range_check_ptr}(
-        user : felt, recipient : felt) -> (claimed : Uint256):
-    Ownable_only_owner()
-    return claimable_claim_rewards(user, recipient)
+        user : felt, recipient : felt) -> (success : felt):
+    let (caller) = get_caller_address()
+
+    with_attr error_message("user should be {caller}"):
+        assert caller = user
+    end
+    let (rewards) = claimable_claim_rewards(user)
+    let (l2_token_bridge_) = Ownable_get_owner()
+    ITokenBridge.mint_rewards(l2_token_bridge_, recipient, rewards)
+    return (TRUE)
```

Comment:
In cairo reverts can't be caught so if the above assert fails the whole contract will. Returning a boolean is an Openzeppelin thing to make sure the call signature is the same as the ERC20 standard but it's unnecessary here.

Reply:
noted